config/environments/production.rb

```diff
@@ -18,14 +18,17 @@
   # Full error reports are disabled and caching is turned on.
   config.consider_all_requests_local = false
   config.action_controller.perform_caching = true
-  # Render exceptions via ErrorsController
+  # Render exceptions via ErrorsController.
   config.exceptions_app = routes
 
-  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
-  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # rubocop:disable Layout/LineLength
+  # Ensures that a master key has been made available in ENV["RAILS_MASTER_KEY"], config/master.key, or an environment
+  # key such as config/credentials/production.key. This key is used to decrypt credentials (and other encrypted files).
+  # rubocop:enable Layout/LineLength
   config.require_master_key = !IS_DOCKER_BUILD
 
-  config.public_file_server.enabled = true
+  # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
+  config.public_file_server.enabled = false
```

Comment:
This change was not prompted by `app:update`, but is just something I noticed that I think should be changed, since we have NGINX serving `public/` assets.

Reply:
I think I shouldn't have made this change. I reverted it here: https://github.com/davidrunger/david_runger/pull/4886 .